ftw/contentpage/profiles/default/jsregistry.xml

```diff
@@ -8,4 +8,9 @@
 
  <javascript cacheable="True" compression="safe" cookable="True"
     enabled="True" expression="" id="++resource++ftw.contentpage.resources/toggle_faq.js" inline="False"/>
+
+ <javascript cacheable="True" compression="safe" cookable="True"
```

Comment:
I've discussed this. It seems that helper_js does not work with resources; the JS must be in a skins directory. :confused: